config-repo/namespaces/boa/balance-reader.yaml

```diff
@@ -29,21 +29,21 @@ spec:
       terminationGracePeriodSeconds: 5
       containers:
       - name: balancereader
-        image: gcr.io/bank-of-anthos/balancereader:v0.3.0
+        image: gcr.io/bank-of-anthos-ci/balancereader:v0.5.2
         volumeMounts:
         - name: publickey
           mountPath: "/root/.ssh"
           readOnly: true
         env:
         - name: VERSION
-          value: "v0.3.0"
+          value: "v0.5.2"
         - name: PORT
           value: "8080"
         # toggle Cloud Trace export
         - name: ENABLE_TRACING
-          value: "false"
+          value: "true"
```

Comment:
I remember that initially the ASD did not want to enable Google Cloud Tracing, Monitoring, etc. in order to reduce demo costs. 
It was actually the reason we added these toggles in the first place.  
Did you intend this change?

Reply:
Thanks for reminding me this historical info. Turned them to false now.